lib/menu.js

```diff
@@ -6,51 +6,238 @@ class PrismarineMenu extends LitElement {
     super()
 
     this.server = config.defaultHost
+    this.serverport = config.defaultHostPort ?? 25565
     this.proxy = config.defaultProxy
+    this.proxyport = !config.defaultProxy && !config.defaultProxyPort ? '' : config.defaultProxyPort ?? 443
     this.username = window.localStorage.getItem('username') ?? 'pviewer' + (Math.floor(Math.random() * 1000))
     this.password = ''
   }
 
   static get properties () {
     return {
       server: { type: String },
+      serverport: { type: Number },
       proxy: { type: String },
+      proxyport: { type: Number },
       username: { type: String },
       password: { type: String }
     }
   }
 
   static get styles () {
     return css`
-        label {
-            grid-column: labels;
-            grid-row: auto;
-        }
+    html {
```

Comment:
html is outside of the web component, you cannot affect it
You can use :host instead if you want to affect the host element